app/views/queue_items/index.html.haml

```diff
@@ -4,31 +4,32 @@
       %article
         %header
           %h2 My Queue
-        %table.table
-          %thead
-            %tr
-              %th(width="10%") List Order
-              %th(width="30%") Video Title
-              %th(width="10%") Play
-              %th(width="20%") Rating
-              %th(width="15%") Genre
-              %th(width="15%") Remove
-          %tbody
-            - @queue_items.each do |queue_item|
+        = form_tag '/update_queue_items', method: :post do
```

Comment:
You should always use named routes, which makes it much easier to modify a url without having to do "shotgun surgery" (which means to have to change code all over because you changed one thing).